packages/workflow/src/Interfaces.ts

```diff
@@ -2166,3 +2166,5 @@ export interface IN8nUISettings {
 		dismissed: string[];
 	};
 }
+
+export type Banners = 'V1' | 'TRIAL_OVER' | 'TRIAL';
```

Comment:
we really should setup a new package for sharing code between FE and BE. `n8n-workflow` is trying to do too many things.